Documentation/tls/tls-certificates.md

```diff
@@ -0,0 +1,49 @@
+# Transport Layer Security (TLS) Certificates
+
+Tectonic secures all communications between cluster components. At install time, certificates can be provided in the following ways, from most automated to most customized:
```

Comment:
Tectonic will ensure that Kubernetes is configured to use TLS to secure all communications between Kubernetes Components. By default we will generate these certificates. These defaults can be overridden using the following methods.